packages/components/src/components/contract-card/contract-card-items/contract-card-header.tsx

```diff
@@ -69,11 +70,12 @@ const ContractCardHeader = ({
     const is_smarttrader_contract = isSmartTraderContract(contract_type);
     const is_mobile = isMobile();
     const is_turbos = isTurbosContract(contract_type);
+    const is_lookbacks = isLookBacksContract(contract_type);
 
     const contract_type_list_info = React.useMemo(
         () => [
             {
-                is_param_displayed: multiplier,
+                is_param_displayed: multiplier && !is_lookbacks,
```

Comment:
In contract card header for `Multipliers` near trade type name we are showing _'X [chosen value of multiplier, e.g.10]'_.  `multiplier` is taken from `contract_info` object.
The issue is that `Lookbacks` contract also includes this field in `contract_info`, but we don't want to show this information in contract card header as it was done for `Multipliers` contract.